### PR TITLE
fix: build-kite mysql_isolated

### PR DESCRIPTION
**Pull request by williamluke4**

### CODE REVIEW

File: src/docker/docker-compose.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ services:
   # - TEST_POSTGRES_URI=postgres://prisma:prisma@postgres/tests
   # - TEST_POSTGRES_ISOLATED_URI=postgres://prisma:prisma@postgres_isolated:5435/tests
   # - TEST_MYSQL_URI=mysql://prisma:prisma@mysql:3306/tests
+  # - TEST_MYSQL_ISOLATED_URI=mysql://prisma:prisma@mysql_isolated:3306/tests
```
Comment (Jolg42) [inline src/docker/docker-compose.yml:8]: this should be removed @williamluke4 this var is not needed there

File: src/docker/docker-compose.yml [hunk 2]
```diff
@@ -13,6 +14,7 @@ services:
   # - TEST_POSTGRES_URI=postgres://prisma:prisma@localhost:5432/tests
   # - TEST_POSTGRES_ISOLATED_URI=postgres://prisma:prisma@localhost:5435/tests
   # - TEST_MYSQL_URI=mysql://prisma:prisma@localhost:3306/tests
+  # - TEST_MYSQL_ISOLATED_URI=mysql://prisma:prisma@localhost:3307/tests
```
Comment (Jolg42) [inline src/docker/docker-compose.yml:17]: this should be removed @williamluke4 this var is not needed there

File: src/docker/docker-compose.yml (hunk 1 above)
Comment (williamluke4) [reply src/docker/docker-compose.yml:8]: Aren't those just comments

Comment (Jolg42) [reply src/docker/docker-compose.yml:8]: Ok I see that the isolated mysql was also added before ✅

**Pull request merged**